src/Artsy/Router/buildServerApp.tsx

```diff
@@ -28,6 +29,23 @@ import { RouterConfig } from "./"
 import { queryStringParsing } from "./Utils/queryStringParsing"
 import { RenderError, RenderPending, RenderReady } from "./Utils/RenderStatus"
 
+import { ChunkExtractor } from "@loadable/server"
+import { getENV } from "Utils/getENV"
+
+// This is the stats file generated by webpack loadable plugin
+let statsFile
+try {
+  statsFile = path.resolve(
+    process.cwd(),
+    "public/assets",
+    "loadable-stats.json"
+  )
```

Comment:
On the force side of things, when we compile our app Webpack is responsible for creating bundle splits wherever it sees `import()` keyword. Loadable components then uses the stats file to parse which components have been dynamically split and creates a manifest to synchronously load on the server and rehydrate on the client.